src/utils/fiatToCrypto.js

```diff
@@ -23,20 +23,30 @@ import { getEnv } from 'configs/envConfig';
 
 import type { AltalixTrxParams, SendwyreRates, SendwyreTrxParams } from 'models/FiatToCryptoProviders';
 import type SDKWrapper from 'services/api';
+import { RAMP_CURRENCY_TOKENS } from '../configs/rampConfig';
 
-export function rampWidgetUrl(address: string, email?: string, plrMode?: boolean) {
+const PILLAR = 'Pillar';
+
+export function rampWidgetUrl(
+  address: string,
+  email?: string,
+  fiatCurrency: string,
+  fiatValue: string,
+) {
   const params = {
-    swapAsset: plrMode ? 'PLR' : null, // This turns on the ability to purchase PLR
+    hostAppName: PILLAR,
+    fiatCurrency,
+    fiatValue,
     hostApiKey: getEnv().RAMPNETWORK_API_KEY,
     userAddress: address,
     ...(email ? { userEmailAddress: email } : {}),
+    swapAsset: RAMP_CURRENCY_TOKENS.join(','),
   };
 
-  return `${getEnv().RAMPNETWORK_WIDGET_URL}?${querystring.stringify(params)}`;
+  return `${getEnv().RAMPNETWORK_WIDGET_URL}?${querystring.stringify(params).replace(/%2C/g, ',')}`;
```

Comment:
I've run the code locally and it seems to work fine without replace.
```suggestion
  return `${getEnv().RAMPNETWORK_WIDGET_URL}?${querystring.stringify(params)}`;
```

Reply:
The ramp url generated without replace  “https://buy.ramp.network/?hostAppName=Pillar&fiatCurrency=GBP&fiatValue=1&hostApiKey=tfekhbxf89587wv4f6g7phdm3wnwsafc8p6pbue2&userAddress=0xa554Ea5F759a8855Ce64e35e84f46fEA5A0ED4C3&userEmailAddress=divya%40bacancy.com&swapAsset=ETH&swapAsset=DAI&swapAsset=PLR&swapAsset=USDC&swapAsset=USDT&swapAsset=MATIC&swapAsset=MATIC_DAI&swapAsset=MATIC_USDC&swapAsset=xDAI” 
with & leads to the destination currency only xDAI. None of the others are showing. 
So, We needed ramp url to be in the format "https://ri-widget-staging-kovan.firebaseapp.com/?hostAppName=Pillar&fiatCurrency=GBP&fiatValue=.2&hostApiKey=tfekhbxf89587wv4f6g7phdm3wnwsafc8p6pbue2&userAddress=0xa554Ea5F759a8855Ce64e35e84f46fEA5A0ED4C3&userEmailAddress=divya%40bacancy.com&swapAsset=ETH,DAI,PLR,USDC,USDT,MATIC,MATIC_DAI,MATIC_USDC,xDAI". 
So used this way. Any better way it can be done without using replace?

Reply:
What have worked for me is to keep the `.join(',')` above, and just remove `.replace()`. Could you check if that works for you too?

Reply:
@mdjastrzebski The url generated is working.  @IAmKio as per the changes the URL generated is in the format "https://buy.ramp.network/?hostAppName=Pillar&fiatCurrency=GBP&fiatValue=1&hostApiKey=tfekhbxf89587wv4f6g7phdm3wnwsafc8p6pbue2&userAddress=0xa554Ea5F759a8855Ce64e35e84f46fEA5A0ED4C3&userEmailAddress=divya%40bacancy.com&swapAsset=ETH%2CDAI%2CPLR%2CUSDC%2CUSDT%2CMATIC%2CMATIC_DAI%2CMATIC_USDC%2CxDAI". Is that fine?

Reply:
@mdjastrzebski @divyaBacancy yeah that looks good to me! Expected asset list on Ramp is being displayed 👍